src/Test/Proxy/SymfonyProxy.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the FOSHttpCache package.
+ *
+ * (c) FriendsOfSymfony <http://friendsofsymfony.github.com/>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace FOS\HttpCache\Test\Proxy;
+use Symfony\Component\Filesystem\Filesystem;
```

Comment:
Tests fail because of a missing dependency. I guess we should add it to our `composer.json`?